science/gr-osmosdr/Portfile

```diff
@@ -139,6 +139,11 @@ variant docs description "Install ${name} documentation" {
         port:doxygen \
         path:bin/dot:graphviz
 
+    if {${subport} eq "gr-osmosdr"} {
+        depends_build-append \
+            port:py${active_python_version_no_dot}-six
```

Comment:
need to rev-bump gr-osmosdr subport when adding a dependency. Thx!

Reply:
Even for build dependencies?

Reply:
https://guide.macports.org/chunked/reference.html#reference.keywords

> Some examples of situations in which a port's revision should not be increased:
>* adding a dependency to depends_fetch, depends_extract, depends_build, or depends_test

Reply:
Hmm ... OK yeah I see the point. hmmm ... OK so here's how I see it: If installing `gr-osmosdr +docs` succeeds without this addition -- it just doesn't build all of the docs, or whatever -- then adding it requires a rev-bump. If the build fails, then we don't require the rev-bump since we're fixing something that doesn't build in the first place.

I -think- the build is broken with this variant, so then the rev-bump isn't required. But I'm not certain. I'll test on one of my build systems at work tomorrow. Do you know which way the build is for +docs here?

Reply:
>OK so here's how I see it: If installing gr-osmosdr +docs succeeds without this addition -- it just doesn't build all of the docs, or whatever -- then adding it requires a rev-bump

If someone managed to install the `+docs` variant, that means they already had `pyXX-six` installed and were not using trace mode; their installation should be fine and not missing any docs. If `pyXX-six` was missing or trace mode was used, then the build fails. So the revision does not need to be incremented in either case.

The failure reported at https://trac.macports.org/ticket/61106#comment:7 is for default variants, which includes `+docs`.

Reply:
Gotcha. Makes sense. Thanks!